### PR TITLE
feature/M2-1282-wname-logic

### PR DESCRIPTION
**Pull request by kdanilenka32**

### CODE REVIEW

File: src/modules/Dashboard/features/Applet/AddUser/AddUserForm/AddUserForm.tsx [hunk 1]
```diff
@@ -27,18 +27,22 @@ import {
 } from './AddUserForm.styles';
 import { Fields, fields, defaultValues, langs, roles } from './AddUserForm.const';
 import { AddUserSchema } from './AddUserForm.schema';
-import { AddUserFormProps, FormValues } from './AddUserForm.types';
+import { AddUserFormProps, FormValues, WworkspaceInfo } from './AddUserForm.types';
```
Comment (IhorNesterenko99) [inline src/modules/Dashboard/features/Applet/AddUser/AddUserForm/AddUserForm.tsx:30]: Pls correct typo WworkspaceInfo

Comment (nmialik) [inline src/modules/Dashboard/features/Applet/AddUser/AddUserForm/AddUserForm.tsx:30]: Please fix name for WworkspaceInfo

File: src/modules/Dashboard/features/Applet/AddUser/AddUserForm/AddUserForm.types.ts [hunk 2]
```diff
@@ -9,11 +9,16 @@ export type FormValues = {
...
+  respondents: { label: string; id: string }[];
 };
 
 export type Field = { name: keyof FormValues; options?: Option[] };
 
 export type AddUserFormProps = {
   getInvitationsHandler: () => void;
 };
+
+export type WworkspaceInfo = {
```
Comment (IhorNesterenko99) [inline src/modules/Dashboard/features/Applet/AddUser/AddUserForm/AddUserForm.types.ts:21]: pls correct typo WworkspaceInfo

File: src/modules/Dashboard/features/Applet/AddUser/AddUserForm/AddUserForm.tsx [hunk 3]
```diff
@@ -86,7 +102,7 @@ export const AddUserForm = ({ getInvitationsHandler }: AddUserFormProps) => {
       unregister(nickname);
       unregister(secretUserId);
       unregister(respondents);
-      workspaceNameShowed && register(workspacePrefix, { value: '' });
+      workspaceNameVisible && register(workspacePrefix, { value: workspaceInfo?.name });
```
Comment (IhorNesterenko99) [inline src/modules/Dashboard/features/Applet/AddUser/AddUserForm/AddUserForm.tsx:105]: what if workspaceInfo?.name would return false for register?

File: src/modules/Dashboard/features/Applet/AddUser/AddUserForm/AddUserForm.tsx (hunk 1 above)
Comment (kdanilenka32) [reply src/modules/Dashboard/features/Applet/AddUser/AddUserForm/AddUserForm.tsx]: fixed

File: src/modules/Dashboard/features/Applet/AddUser/AddUserForm/AddUserForm.types.ts (hunk 2 above)
Comment (kdanilenka32) [reply src/modules/Dashboard/features/Applet/AddUser/AddUserForm/AddUserForm.types.ts]: fixed

File: src/modules/Dashboard/features/Applet/AddUser/AddUserForm/AddUserForm.tsx (hunk 3 above)
Comment (kdanilenka32) [reply src/modules/Dashboard/features/Applet/AddUser/AddUserForm/AddUserForm.tsx]: fixed

**Review by nmialik — APPROVED** [review]

**Review by IhorNesterenko99 — APPROVED** [review]

**Review by muhurov — APPROVED** [review]

**Pull request merged**